WalletWasabi.Fluent/Views/Wallets/Home/Tiles/WalletBalanceTileView.axaml

```diff
@@ -10,15 +10,31 @@
              x:DataType="tiles:WalletBalanceTileViewModel"
              x:Class="WalletWasabi.Fluent.Views.Wallets.Home.Tiles.WalletBalanceTileView">
 
-  <c:TileControl Title="BALANCE" IsBottomContentVisible="{Binding HasBalance^}" SeparatorMargin="0 0 0 6">
-    <Viewbox Margin="0 3" VerticalAlignment="Center" MaxHeight="40">
-      <c:PrivacyContentControl Classes="h2 monoSpaced" HorizontalContentAlignment="Center"
-                               Content="{Binding BalanceBtc^}" />
-    </Viewbox>
-    <c:TileControl.BottomContent>
-      <c:PrivacyContentControl Classes="bold monoSpaced" Margin="4" VerticalAlignment="Center"
-                               HorizontalAlignment="Center" Opacity="0.8"
-                               Content="{Binding BalanceFiat^, Converter={x:Static converters:MoneyConverters.ToUsdAproxBetweenParens}}" />
-    </c:TileControl.BottomContent>
-  </c:TileControl>
+  <Grid>
```

Comment:
Grid with no rows / columns, best to use <Panel> in avalonia... much more lightweight.

Reply:
Good catch! I completely missed that one.